pkg/controller/cabundleinjector/validatingwebhook.go

```diff
@@ -0,0 +1,59 @@
+package cabundleinjector
+
+import (
+	"bytes"
+
+	admissionreg "k8s.io/api/admissionregistration/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	admissionregclient "k8s.io/client-go/kubernetes/typed/admissionregistration/v1"
+	admissionreglister "k8s.io/client-go/listers/admissionregistration/v1"
+	"k8s.io/klog"
+)
+
+type validatingWebhookCABundleInjector struct {
+	client   admissionregclient.ValidatingWebhookConfigurationInterface
+	lister   admissionreglister.ValidatingWebhookConfigurationLister
+	caBundle []byte
+}
+
+func newValidatingWebhookInjectorConfig(config *caBundleInjectorConfig) controllerConfig {
+	informer := config.kubeInformers.Admissionregistration().V1().ValidatingWebhookConfigurations()
+	keySyncer := &validatingWebhookCABundleInjector{
+		client:   config.kubeClient.AdmissionregistrationV1().ValidatingWebhookConfigurations(),
+		lister:   informer.Lister(),
+		caBundle: config.caBundle,
```

Comment:
Could you do V1betav1 as well? V1 is not supported yet by apimachinery, so operator building on top of that (anything built with operator-sdk) can't use this yet.

Reply:
A wise person reminded me that v1beta1 is supported on a server that supports v1 thanks to the magic of automatic conversion. So this will work regardless.

Reply:
alright! thanks!